### PR TITLE
add an example how to use routegroups in stackset

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

File: docs/howtos.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ spec:
         spec:
           containers:
           - name: skipper
-            image: registry.opensource.zalan.do/pathfinder/skipper:v0.11.194
+            image: registry.opensource.zalan.do/pathfinder/skipper:latest
```
Comment (mikkeloscar) [inline docs/howtos.md:37]: This was also changed to `teapot` instead of `pathfinder` right?

Comment (szuecs) [reply docs/howtos.md:37]: oh, right

Comment (szuecs) [reply docs/howtos.md]: fixed

**Comment by mikkeloscar** [comment]
:+1:

**Comment by jonathanbeber** [comment]
:+1:

**Pull request merged**